features/facebookImport/src/components/dataStructureMiniStory/dataStructureMiniStory.jsx

```diff
@@ -84,12 +83,15 @@ const DataStructureMiniStory = ({ data }) => {
                     onBubbleClick={handleBubbleClick}
                 />
             </div>
-            <ChartButtons
-                buttonsContent={dataWithTotal.map((d) => {
+            <Chips
+                chipsContent={dataWithTotal.map((d) => {
                     return { id: d.title };
                 })}
                 activeButton={selectedFolder}
                 onButtonsClick={handleFolderSelected}
+                activeChip={selectedFolder}
+                onChipClick={handleFolderSelected}
+                theme={"dark"}
```

Comment:
So, possibly I'm crazy, but I'm wondering: We do need themes to _cascade_ -> We don't really want to set `theme` for each component individually I suppose.

We could sure implement that, but I was wondering: Why not just use CSS for that? It already cascades. AFAIK, if we set the class name to `poly-theme-dark` in a top level component, and then `poly-theme-light` in a detail component, it should work as expected - that detail component having the light theme. That one, and all of its children - that _appears_ to be exactly the behaviour we need, looking at the cluster stories for example.

Maybe too radical of an idea to start with though.

Reply:
Oh sorry, got lost in this idea and forgot to be concrete :D Two things:

1. Instead of having a `theme` attribute here, you simply set `className` (and ensure it ends up in the generated div - not sure if that's default behaviour in React, probably not. In that case it'd be the same amount of work to expose `className`, and then I'd personally probably prefer `theme`, the way it is.)
2. In order to have a default theme, you could simply do a selector like `:root, .poly-theme-light {` for the default theme.

Reply:
Na you're totally right. This is what I had in mind designing this. Eg. in fbImport setting dark on main and light on HTRT for example